packages/site/src/pages/index.tsx

```diff
@@ -36,148 +51,418 @@ const features = [
 
 export default function App() {
   return (
-    // @ts-ignore
-    <ThemeProvider theme={theme}>
-      <Stack axis="y" width="100%" paddingX="10rem" paddingY="6rem">
-        <header style={{ display: 'grid', gridTemplateColumns: '1fr 1fr' }}>
-          <img src={logoSrc} alt="logo" width="100px" />
-          <span style={{ justifySelf: 'end' }}>
-            <Stack as="nav" axis="x" gap={8}>
-              <GitHubIcon />
-            </Stack>
-          </span>
-        </header>
-
-        <Spacer.Vertical size="4rem" />
-
-        <Stack axis="x" justifyContent="fill">
-          <Stack axis="y" gap="2rem">
-            <Text variant="heading1">
-              Build Your Own <Text variant="mark">Design Tool</Text>
-            </Text>
-            <Text variant="body1" width="24ch">
-              Noya is an ecosystem of design utilities to help with your next
-              project.
-            </Text>
-          </Stack>
-          <Stack axis="y" alignItems="center" justifyContent="center">
-            Demo Here
+    <Layout>
+      <Spacer size="1rem" />
+
+      <Stack
+        as="header"
+        gridColumn="1 / -1"
+        flexDirection="row"
+        justifyContent="space-between"
+      >
+        <Stack
+          as="img"
+          src={logoSrc}
+          alt="logo"
+          height="var(--header-icon-size)"
+        />
+        <Stack as="nav" flexDirection="row" gap={8}>
+          <Stack as="a" href="https://github.com/noya-app/noya">
+            <GitHubIcon />
           </Stack>
         </Stack>
+      </Stack>
 
-        <Spacer.Vertical size="12rem" />
-
-        <Stack axis="x" alignItems="center" justifyContent="fill" gap="2rem">
-          <Grid columns="1fr 1fr" gap="1rem">
-            {features.map((feature) => (
-              <Stack key={feature.name} axis="x" padding="1rem">
-                <Text variant="body1">{feature.name}</Text>
-              </Stack>
-            ))}
-          </Grid>
-          <Stack axis="y" gap="2.5rem">
-            <Text variant="heading2">
-              All Of The Features You’re Used To{' '}
-              <Text variant="mark">{preventTextOprhan('And More')}</Text>
-            </Text>
-            <Text variant="body1">
-              Full-featured while delivering a system that can cater to your
-              specific needs.
-            </Text>
-          </Stack>
+      <Spacer
+        css={{
+          minHeight: '6rem',
+          [mediaQuery.medium]: {
+            minHeight: '8rem',
+          },
+        }}
+      />
+
+      <Grid
+        gridColumn="1 / -1"
```

Comment:
Style props fall apart a little here with responsive. We could do something like an object:
```jsx
<Grid column={{ initial: '1 / -1', medium: '2 / span 10'  }} />
```
But I didn't have time to spend on that. This was something I solved recently in JSXUI I'd love to discuss further.